ec2/tags.go

```diff
@@ -19,6 +20,43 @@ func (e *Ec2) UpdateTags(ctx context.Context, rawTags map[string]string, ids ...
 		tags = append(tags, &ec2.Tag{Key: aws.String(key), Value: aws.String(val)})
 	}
 
+	instanceIDs := []*string{}
+	for _, id := range ids {
+		if strings.HasPrefix(id, "i-") {
+			instanceIDs = append(instanceIDs, aws.String(id))
+		}
+	}
+
+	describeVolumesInput := ec2.DescribeVolumesInput{
```

Comment:
I don't think all this logic belongs here - should probably move to an orchestrator function.

Reply:
We already have a `ListInstanceVolumes` function that does this, so why not just call that?